### DISCUSSION TITLE
Link rot

### DISCUSSION THREAD

**Discussion by codingthat** [legendary-gl/legendary#369] [General]
## Expected Behavior

The link at https://github.com/derrod/legendary/wiki/Available-Linux-Packages#debianubuntu-dur should do what it says on the tin.

## Current Behavior

It redirects to a personal home page.

**Comment by codingthat** [comment]
Furthermore, the instructions just beneath it fail along similar lines:

```
$ git clone https://dur.hunterwittenborn.com/legendary.git
Cloning into 'legendary'...
fatal: unable to update url base from redirection:
  asked for: https://dur.hunterwittenborn.com/legendary.git/info/refs?service=git-upload-pack
   redirect: https://www.hunterwittenborn.com/
```

**Comment by derrod** [comment]
Thanks for the heads-up. I removed the old entry.
